cmd/csi-node-driver-registrar/main.go

```diff
@@ -41,6 +42,20 @@ const (
 	sleepDuration = 2 * time.Minute
 )
 
+const (
+	// ModeDefault is the default running mode of node-driver-registrar
+	ModeDefault = "default"
+
+	// ModeKubeletRegistrationProbe makes node-driver-registrar act as an exec probe
+	// that checks if the kubelet plugin registration succeded.
+	ModeKubeletRegistrationProbe = "kubelet-registration-probe"
+)
+
+var (
+	// The kubelet registration lockfile path, set when the program runs
+	kubeletRegistrationLockfilePath = ""
```

Comment:
nit: we're not really locking anything here. Can we call this something like "registration probe"?